projectile.el

```diff
@@ -2432,6 +2433,8 @@ TEST-DIR which specifies the path to the tests relative to the project root."
                                   :compile "boot aot"
                                   :test "boot test"
                                   :test-suffix "_test")
+(projectile-register-project-type 'clj '("deps.edn")
```

Comment:
Technically speaking this is not a `clj` project, but `clojure-cli` or `clojure-tools` depending on your perspective. In CIDER I've named it `clojure-cli`.